source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/project-directory/show/index.tsx

```diff
@@ -0,0 +1,4 @@
+
+export const pathName = '/directory/:projectId';
+
+export default props => "TBD";
```

Comment:
is this supposed to be the public view of a project?

Reply:
Correct.